src/org/labkey/targetedms/view/pharmacokinetics.jsp

```diff
@@ -82,7 +82,8 @@
                 <span id="nonIVC0Controls-Warn-<%=h(subgroup)%>" class="labkey-error"><h4>WARNING: Please enter a non-IV C0 and recalculate.</h4></span>
                 non-IV C0
                 <input type="number" id="nonIVC0-<%=h(subgroup)%>" label="non-IV C0"/>
-                <button id="btnNonIVC0-<%=h(subgroup)%>" onclick="updateStatsForNonIVC0('<%=h(subgroup)%>')">Recalculate</button>
+                <% addHandler("btnNonIVC0-"+h(subgroup), "click", "updateStatsForNonIVC0('"+ h(subgroup) + "')"); %>
```

Comment:
Similar here - probably don't need the call to `h()` or we'll get double-encoding